### DISCUSSION TITLE
Wrong Date on Images

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#2404] [Q&A]
I have an issue with images, particularly from WhatsApp, when they are imported into photoprism. The filename has the month and the date in it, and my computer file manager picks up the date and time the image was taken. When I upload to photoprism, they all just get dated as 00:00 on the first of the month. Is there a fix for this (sans manually changing all of the dates in photoprism). i just started using the software so im fairly new but I really would like to figure out how to solve this issue before I dump thousands of photos into it.

image in dolphin file manager (localed in ~/Pictures/2022/05/):
![image](https://user-images.githubusercontent.com/47161945/172032890-3deeb8e3-f7a0-41c4-a04c-df038da01247.png)

same image in photoprism:
![image](https://user-images.githubusercontent.com/47161945/172032948-0aef1534-2412-4396-9547-666646f42721.png)

**Comment by graciousgrey** [comment]
We have an open issue to support reading dates from Whats App filenames

https://github.com/photoprism/photoprism/issues/1102
